Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs

```diff
@@ -63,6 +63,11 @@ internal async static Task<List<Result>> ExecuteWindowsIndexSearchAsync(string i
                     }
                 }
             }
+            catch (OperationCanceledException)
+            {
+                // return empty result when cancelled
+                return results;
+            }
```

Comment:
Previously OperationCanceledException is thrown as normal exception, causing the slowness of logging exception.